Dockerfiles/agent/entrypoint/50-kubernetes.sh

```diff
@@ -7,7 +7,7 @@ fi
 # Set a default config for Kubernetes if found
 # Don't override /etc/datadog-agent/datadog.yaml if it exists
 if [[ ! -e /etc/datadog-agent/datadog.yaml ]]; then
-    if [[ -e /var/run/docker.sock ]]; then
+    if [[ -S /var/run/docker.sock ]] || ( [[ "$DD_CRI_SOCKET_PATH" =~ docker\.sock ]] && [[ -S "$DD_CRI_SOCKET PATH" ]] ); then
```

Comment:
You miss an `_` in the last `$DD_CRI_SOCKET PATH`

Reply:
Fixed!